pkg/strategy/autobuy/strategy.go

```diff
@@ -64,27 +65,57 @@ func (s *Strategy) InstanceID() string {
 }
 
 func (s *Strategy) Validate() error {
+	if s.Symbol == "" {
+		return fmt.Errorf("symbol is required")
+	}
+
+	if s.Schedule == "" {
+		return fmt.Errorf("schedule is required")
+	}
+
+	if s.MinBaseBalance.Sign() <= 0 {
+		return fmt.Errorf("minBaseBalance must be greater than 0")
+	}
+
 	if err := s.QuantityOrAmount.Validate(); err != nil {
 		return err
 	}
+
+	if s.Bollinger != nil {
+		if s.Bollinger.Interval == "" {
+			return fmt.Errorf("bollinger interval is required")
+		}
+
+		if s.Bollinger.BandWidth <= 0 {
+			return fmt.Errorf("bollinger band width must be greater than 0")
+		}
+	}
 	return nil
 }
 
 func (s *Strategy) Defaults() error {
+	if s.OrderType == "" {
```

Comment:
order type needs to be aligned with the price type

if price type = taker, the default limit maker won’t work

Reply:
so add a warning here?

Reply:
the default order type should be "limit", because it would work for all the cases

Reply:
I will set the default order type to limit